### DISCUSSION TITLE
Send Later returns "unknown error" when delaying send

### DISCUSSION THREAD

**Discussion by wilsonb2** [Extended-Thunder/send-later#432] [Q&A]
I've been getting this error when I try to use Send Later and send other then "Send Now":

Sending of the message failed. The message could not be sent using Outgoing server (SMTP) smtp.goglemail.com for an unknow reason". 

Send later has work great for a number of years. This just started.

I'm using TB Ver. 91.7.0 (64-bit). It may have something to do with my settings since it has worked so well for so long. I just reinstalled it though and maybe something happened then.

Any ideas as to what I should try or to look?  Or maybe you need to see my setup files? When I click "Send Now", all works well. So, there is something in the delay that's causing the issue. The email show in the "Sent" folder and seems to actually send.

Thank you for any help you can give!

**Comment by jph-sendlater** [comment]
Try the following steps:
Compose an email to yourself
Use the "Put in Outbox" option
Verify that the message shows up in `Local Folders / Outbox`
Use `File > Send Unsent Messages` to manually trigger sending messages from the outbox

That's just manually stepping through the process that send later would use, so I would expect the same error dialog. Let me know if the same thing happens, or if the message sends without error.

**Comment by wilsonb2** [comment]
I did get the same error message. But the email was sent and the outbox is
now empty.

Thanks for working with me on this.

Best Regards,

Wilson J. Bailey, P.E.
281.520.0336

**Comment by wilsonb2** [comment]
I thought about the possibility of the Local Folder Outbox being 
corrupted. I assume by using Send Now, Send Later bypassed the outbox. 
Just a thought.

Best Regards,

Wilson J. Bailey, P.E.
(281) 520-0336

**Comment by wilsonb2** [comment]
I'll just use gMail and Boomerang until this gets worked out.

Best Regards,

Wilson J. Bailey, P.E.
(281) 520-0336

**Comment by wilsonb2** [comment]
I tried the same thing on another computer with tbird and send-later on 
it. I worked fine. (Unlike previous problems, both were open the the 
same email account.) Now, it seems to be working on both computers.

Computers do seem to have a mind of their own!

Best Regards,

Wilson J. Bailey, P.E.
(281) 520-0336
